packages/splits-sdk/src/client/splitV2.ts

```diff
@@ -298,13 +300,27 @@ class SplitV2Transactions extends BaseTransactions {
   ): Promise<{ split: Split }> {
     const publicClient = this._getPublicClient(chainId)
 
-    const [createLogs, owner, paused] = await Promise.all([
+    const [createV2o1Logs, createV2Logs, owner, paused] = await Promise.all([
+      publicClient.getLogs({
+        address: [
+          getSplitV2o1FactoryAddress(chainId, SplitV2Type.Pull),
+          getSplitV2o1FactoryAddress(chainId, SplitV2Type.Push),
+        ],
+        event: splitV2o1CreatedEvent,
+        args: {
+          split: splitAddress,
+        },
+        strict: true,
+        fromBlock: getSplitV2FactoriesStartBlock(chainId),
+      }),
       publicClient.getLogs({
-        event: splitCreatedEvent,
         address: [
           getSplitV2FactoryAddress(chainId, SplitV2Type.Pull),
           getSplitV2FactoryAddress(chainId, SplitV2Type.Push),
+          getSplitV2o1FactoryAddress(chainId, SplitV2Type.Pull),
+          getSplitV2o1FactoryAddress(chainId, SplitV2Type.Push),
```

Comment:
are these supposed to be here? shouldn't this just be the splitv2 (not 2.1)?

Reply:
we have two create events in v2.1, one of them is identical to the v2 create event so I clubbed it in one search

Reply:
i see. ok cool. maybe just a comment about that.